hypothesis-python/docs/how-to/suppress-healthchecks.rst

```diff
@@ -0,0 +1,31 @@
+How to suppress a health check everywhere
+=========================================
+
+Hypothesis sometimes raises a |HealthCheck| to indicate that your test may be less effective than you expect, slower than you expect, unlikely to generate effective examples, or otherwise has silently degraded performance.
+
+While |HealthCheck| can be useful to proactively identify issues, you may not care about certain classes of them. If you want to disable a |HealthCheck| everywhere, you can define and load a :ref:`settings profile <settings_profiles>`. Place the following code in your ``conftest.py`` (or in any file which pytest loads during collection):
```

Comment:
Happy to present pytest as the default option; don't want to assume that you're necessarily using it - PSF survey is 52% pytest, 25% unittest users (with some overlap of course, there's 36% not testing)